### PR TITLE
Update jquery qtip to v2.2.0.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (haacked) [inline _includes/header.html:5]: This change seems unnecessary.

File: css/application.css [hunk 2]
```diff
@@ -451,6 +455,8 @@ strong {
   font-size: 90%;
 }
 
+{% include css/jquery.qtip.min.css %}
```
Comment (haacked) [inline css/application.css:458]: Why this change? Just curious.

Comment (XhmikosR) [inline css/application.css:458]: With this, we include jquery.qtip.min.css in application.css when jekyll is run. Thus we have one less http request since we only need to include application.css.

File: _includes/header.html (hunk 1 above)
Comment (XhmikosR) [inline _includes/header.html:5]: Yeah, slipped into. I can revert it make it in a separate commit or just go along with it since it's purely cosmetic :P

**Comment by haacked** [comment]
thumbs-up-computer-kid-gif
